.github/workflows/build.yml

```diff
@@ -21,9 +24,6 @@ jobs:
           path: ~/go/pkg/mod
           key: go-${{ hashFiles('**/go.sum') }}
```

Comment:
`action/checkout` should be taken place before the step because if not, there will be no `go.sum`.

Reply:
Thanks for your help!!